src/oc/web/components/ui/emoji_picker.cljs

```diff
@@ -136,23 +136,24 @@
                                                 @caret-pos)
                                             (not @visible))]
                                (reset! visible vis)))}]
-      [:div.picker-container
-        {:class (utils/class-set {position true
-                                  :preloading (not @(::preloaded s))
-                                  :visible @visible})}
-        (when-not (utils/is-test-env?)
-          (react-utils/build (.-Picker js/EmojiMart)
-           {:native true
-            :onClick (fn [emoji event]
-                      (when (and default-field-selector
-                                 (not @(::caret-pos s)))
-                        (utils/to-end-of-content-editable (.querySelector js/document default-field-selector))
-                        (save-caret-position s))
-                      (let [add-emoji? (boolean @(::caret-pos s))]
-                         (when add-emoji?
-                           (replace-with-emoji caret-pos emoji)
-                           (remove-markers s)
-                           (.focus @last-active-element))
-                         (reset! visible false)
-                         (when (fn? add-emoji-cb)
-                           (add-emoji-cb @last-active-element emoji add-emoji?))))}))]]))
\ No newline at end of file
+     (when @visible
+       [:div.picker-container
+         {:class (utils/class-set {position true
+                                   :preloading (not @(::preloaded s))
+                                   :visible @visible})}
```

Comment:
I think we can remove this `visible` class since we hide the picker w/o rendering it at all now. Before it was always rendered inside the page even if it was hidden for most of the time.

Reply:
Also ::preloading is not used anymore since it only renders when it's needed, we can remove all that code too.